frontend/model.ts

```diff
@@ -0,0 +1,18 @@
+export interface fill_project {
+    image: string;
```

Comment:
Can't you have multiple files ?

Reply:
Should I put "User" and "Project" in two different files?

Reply:
No...